end-to-end/walkthrough.md

```diff
@@ -0,0 +1,769 @@
+# Attestation Walkthough
+
+This guide explains how to setup a verifier and an attester. To produce a workable example setup, it is necessary to 
+
+- create attestation keys on the attester,
+- provisioning trust anchors, reference values and endorsements on the verifier, and
+- to create evidence on the attester for consumption by the verifier.
+
+For this example setup evidence is provided in form of an PSA attestation token.
+
+## Software Dependencies
+
+The software depends at least on 
+
+- Docker X, and
+- Go
+
+Veraison also has additional dependencies, which are described at https://github.com/veraison/services
+
+Important: Do not install Docker Desktop (at least on Ubuntu) since otherwise Veraison will fail.
+
+Use the following command to install the dependencies for Veraison on an Ubuntu OS:
+
+~~~
+sudo apt install git docker.io jq tmux docker-buildx
+~~~
+
+## Veraison
+
+This section provides details about the use of the Veraison docker image, which is meant to be used for demo purporses. The repository of Veraison can be found at https://github.com/veraison/services
+
+In summary, the following steps need to be executed:
+
+~~~
+git clone https://github.com/veraison/services.git
+cd services
+make docker-deploy
+
+source deployments/docker/env.bash
+veraison status
+~~~
+
+If Veraison services are not running, then they need to be started with the following command:
+
+~~~
+veraison start
+~~~
+
+To check the status of the database use the following command:
+
+~~~
+veraison check-stores
+~~~
+
+Of course, after the initial installation the database will be empty. Provisioning of reference values, trust anchors and endosements will
+be explained below and those will be stored in the database. If you haven't created attestation keys yet, then first look at the next section otherwise skip it. 
+
+In case of problems, enable debugging by issuing the following command in a separate command line window. 
+
+~~~
+veraison follow <service-name>
+~~~
+
+Replace <service-name> with one of the services available with Veraison, namely vts, provisioning, verification, management, or keycloak.
+
+Sometimes it is necessary to remove the content of the database. In this case run the following command. Note that this requires you to re-run 'make docker-deploy'!
```

Comment:
`make really-clean` wil completely dismantle the deployment and remove the build artifacts. If the goal is to clear the contents of the stores, this can be done with `veraison clear-stores` without  removing the deployment or stopping the services.